.gitignore

```diff
@@ -22,3 +22,4 @@ coverage.xml
 .vs/
 decksite/images/*-cmc.png
 .web_cache/
+my_doodle.py
```

Comment:
my_doodle.py is where I put my 5 line scripts to test out things. Because I often need to import things (to play around with them), I put it in this folder, but because I didn't want to push it, I put it in gitignore. Probably not very pretty.